packages/frontend/src/app/page.tsx

```diff
@@ -20,12 +24,15 @@ async function getEntities(): Promise<Entity[]> {
 
 export default async function Home() {
   const entities: Entity[] = await getEntities()
+  const token = getAuthToken()
+  const user = await getUser(token || '')
+  console.log('user', user)
 
   return (
     <PageContainer>
       <PageHeading>
         <Text4xlBold>Trusted Entities</Text4xlBold>
-        <Button title="Invite a company" />
+        {user.user && <Button title="Invite a company" />}
```

Comment:
I don't like `user.user`, either we can change the data returned from backend API to return id property, and then we can reference `user.id`, or we just return the string without an object. Then it could be `getUserId` etc.

Reply:
I changed it to `user.id`.